src/features/user/MFV2/MyContributions/DonateButton.tsx

```diff
@@ -18,6 +18,7 @@ type DonateButtonProps = (SupportedDonationButton | RegularDonationButton) & {
   projectSlug: string;
   contributionUnitType: 'tree' | 'm2';
   buttonText: string;
+  customClass?: string;
 };
```

Comment:
While `customClass` isn't needed any more, it can be left in `DonateButtonProps` the make it easy to customize the button.